themes.md

```diff
@@ -0,0 +1,81 @@
+The bar can be themed either by specifying a pre-complied theme or overwriting defaults in the configuration.  
+We differentiate between themes and icon sets.
+
+In order to change either, add them to your configuration:
+
+```toml
+theme = "solarized-dark"
+icons = "awesome"
+```
+
+Available themes are: `plain`, `solarized-dark` and `slick`.  
+Available icon sets are: `none`, `awesome`, `material`.
+
+> **Note**: In order to use the material icon set, you need a patched material icons font which can be found [here](https://gist.github.com/draoncc/3c20d8d4262892ccd2e227eefeafa8ef/raw/3e6e12c213fba1ec28aaa26430c3606874754c30/MaterialIcons-Regular-for-inline.ttf). Make sure to pass it in your i3 configuration bar block.
+
+## Overwriting themes and icon sets
+Create a block in the configuration called theme or icons like so:
+
+```toml
+[theme]
+name = "solarized-dark"
+[theme.overrides]
+idle_bg = "#123456"
+idle_fg = "#abcdef"
+
+[icons]
+name = "awesome"
+[icons.overrides]
+bat = " | | "
+bat_full = " |X| "
+bat_charging = " |^| "
+bat_discharging = " |v| "
+```
+
+Example configurations can be found as `example_theme.toml` and `example_icon.toml`.
+Here's a full list of available overrides:
+
+| Theme        		    | Icons
```

Comment:
Can you just turn this into two bullet lists? Maintaining a table is going to turn into a whitespace nightmare.

Reply:
What do you think about a preparing a small script to generate the table from the respective source files? Would eliminate mistakes plus we could keep the table formatting.

Reply:
For now I will change it to two bullet lists.